nixos/modules/services/desktops/system76-scheduler.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, ... }:
+
+let
+  cfg = config.services.system76-scheduler;
+
+  readDefaultConfig = path:
+    lib.fromRON (builtins.unsafeDiscardStringContext (builtins.readFile "${cfg.package}/data/${path}"));
+
+  default = {
+    config = readDefaultConfig "config.ron";
+    assignments = readDefaultConfig "assignments.ron";
+    exceptions = readDefaultConfig "exceptions.ron";
+  };
+in {
+  options = {
+    services.system76-scheduler = {
+      enable = lib.mkEnableOption (lib.mdDoc "system76-scheduler");
+
+      package = lib.mkOption {
+        type = lib.types.package;
+        default = config.boot.kernelPackages.system76-scheduler;
+        defaultText = lib.literalExpression "config.boot.kernelPackages.system76-scheduler";
+        description = lib.mdDoc "Which System76-Scheduler package to use.";
+      };
+
+      niceness = {
+        background = lib.mkOption {
+          type = lib.RON.types.optional lib.types.int;
+          default = default.config.background;
+          defaultText = "<$(dirname $(realpath $(which system76-scheduler)))/../data/config.ron>.background";
+          description = lib.mdDoc "Background niceness.";
+        };
+        foreground = lib.mkOption {
+          type = lib.RON.types.optional lib.types.int;
+          default = default.config.foreground;
+          defaultText = "<$(dirname $(realpath $(which system76-scheduler)))/../data/config.ron>.foreground";
+          description = lib.mdDoc "Foreground niceness.";
+        };
+      };
+
+      exceptions = lib.mkOption {
+        type = lib.types.listOf lib.types.str;
+        default = [];
+        description = lib.mdDoc "Programs that are left untouched; merged with `$(dirname $(realpath $(which system76-scheduler)))/../data/exceptions.ron`.";
+      };
+
+      assignments = lib.mkOption {
+        type = lib.RON.types.map (lib.types.listOf lib.types.str);
+        default = [];
+        description = lib.mdDoc "Assignments of priorities to programs; merged with `$(dirname $(realpath $(which system76-scheduler)))/../data/assignments.ron`.";
+      };
```

Comment:
How about just making this be `assignmentPaths` instead, allowing users to point to a RON file? Then we don't need to be involved in RON at all. In any case, options like these need to link to upstream documentation on how to specify configurations.